packages/contracts/src/contract-data.ts

```diff
@@ -0,0 +1,130 @@
+import * as path from 'path'
+
+export const getL1ContractData = (network: 'goerli' | 'kovan' | 'mainnet') => {
+  return {
+    Lib_AddressManager: require(path.resolve(
+      __dirname,
+      `../deployments/${network}-v2/Lib_AddressManager.json`
```

Comment:
Isn't `-v2` a hack? How do we know this will stay the correct folder name? Does it work locally / against an `l1chain`?

Reply:
fixed by removing the old versions from the directory